### PR TITLE
feat(aio): update the home page and docs landing page content

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: aio/content/navigation.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
     {
       "url": "events",
       "title": "Events"
+    },
+    {
+      "url": "https://blog.angularjs.org/",
+      "title": "Blog"
```
Comment (gkalpak) [inline aio/content/navigation.json]: Shouldn't this be in `TopNarNarrow` as well (to be shown in SideNav on small screens)?

**Comment by gkalpak** [comment]
BTW, I can reproduce the issue locally. I guess it is because if the `PATCH_LOCK` file already exists, the new patches are not applied. We need `PATCH_LOCK` versioning (or delete node_modules every time the patches get updated) :smiley:

**Comment by gkalpak** [comment]
(I have been posting comments on wrong issues - Sorry)
So, this fails on Travis because of old Chromum version.

**Comment by gkalpak** [comment]
(I have been posting comments on wrong issues - Sorry)
So, this fails on Travis because of old Chromum version.

File: aio/content/marketing/docs.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ Angular is a platform that makes it easy to build applications with the web. Ang
 <div class="card-container">
   <a href="generated/live-examples/quickstart/eplnkr.html" target="_blank" class="docs-card"
     title="Experience Angular in a live coding environment">
-      <section>Experience Angular</section>
-      <p>A quick look at an Angular application.</p>
+      <section>Get a Glimps of Angular</section>
```
Comment (robwormald) [inline aio/content/marketing/docs.md]: spelling: Glimpse

Comment (IgorMinar) [inline aio/content/marketing/docs.md]: good catch :)

File: aio/content/navigation.json (hunk 1 above)
Comment (IgorMinar) [inline aio/content/navigation.json]: done

**Review by gkalpak — APPROVED** [review]
LGTM as soon as Travis is happy.

One side effect of this is that the search-box is shorter than it used to be when expanded, because is it a percentage of the remaining top-bar space. Not sure if it needs to be addressed or not and certainly doesn't have to be part of this PR.

**Pull request merged**